packages/@guardian/source-react-components-development-kitchen/src/lines/SquigglyLines.tsx

```diff
@@ -0,0 +1,78 @@
+import type { SerializedStyles } from '@emotion/react';
+import type { EmotionJSX } from '@emotion/react/types/jsx-namespace';
+import { breakpoints } from '@guardian/source-foundations';
+import type { LineCount } from './Lines';
+
+const wavelength = 12;
+const amplitude = 3;
+const thickness = 1;
+const gap = 3;
+const squiggliness = wavelength / 8;
+const maxWidth = breakpoints.wide;
+
+export const getHeight = (count: LineCount): number =>
+	thickness + gap * Math.max(count, 1);
+
+const d = [
+	`M 0 ${thickness / 2}`,
+	`q ${squiggliness} 0 ${wavelength / 4} ${amplitude / 2}`,
+	`t ${wavelength / 4} ${amplitude / 2}`,
+	`t ${wavelength / 4} -${amplitude / 2}`,
+	`t ${wavelength / 4} -${amplitude / 2}`,
+	`t 12 0`,
```

Comment:
This has led me down a really interesting deep dive into how Bézier curves work, thank you 🙂.

I have some questions. None of them are blockers for this PR, more just to have an interesting conversation:

1. I may have misunderstood, but is your amplitude here actually 1.5 and these should all be `amplitude` rather than `amplitude / 2`? The amplitude is the difference between the equilibrium and the maximum value, rather than the difference between the maximum and the minimum. For a cosine wave (which is what I think you're drawing here?):

![amplitude](https://user-images.githubusercontent.com/53781962/162436916-dcc03dc0-373b-4994-8a9a-7648bb227c80.jpg)

2. What does the final `t 12 0` do? Given where you are in the curve at that point I'd expect that to draw a straight line from `(12, 0)` to `(24, 0)`. I think I'm missing something 🤔

3. Are the subsequent `t`s on lines 20, 21, and 22 needed? Can you get by with the first `t` on line 19 and then just coords after that? Or have you included them more for symmetry rather than necessity?

I'm not that knowledgeable about this so I may have misunderstood one or all of the above points 🙈. Let me know what you think 🙂

Reply:
1. Good catch! Will fix. I thought amplitude was from the tip of the mount to the bottom of the valley.
2. Not sure it’s needed, but I think it helped with pattern edged. Probably best to use `stroke-linecap="square"`!
3. There’s only one `q`, `t` are just coords. I don’t think SVGs allow to just string coordinate pairs without the `t` instruction

Reply:
> There’s only one `q`, `t` are just coords. I don’t think SVGs allow to just string coordinate pairs without the t instruction

I think you might be able to do this:

`q` *dx1* *dy1* *dx* *dy* `t` *dx2* *dy2* *dx3* *dy3* *dx4* *dy4*

based on what the docs say about subsequent coordinate pairs for the `t` command:

https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/d#quadratic_b%C3%A9zier_curve